### ISSUE TITLE
Add new settings to determine if analytical PCR results/gel results should alter the queueing of each other... in minipreps.

### ISSUE THREAD

**Issue by evelynqi** [bthuronyi/CloneCoordinate#197]
Right now, if we turn on the settings that require analytical PCR, quant, and gel before sequencing, the miniprep is ready for all of these. If quant shows low concentration, then the status is "Failed: low conc!" and nothing else can proceed. However, if there is a bad gel or analytical PCR, it still asks users to continue the rest of the steps. If there's a bad analytical PCR, it still tells users that the construct is ready for gel, and vice versa. We should implement a setting that allows users to determine when to stop pushing the miniprep through the pipeline.

**Comment by bthuronyi** [comment]
Agreed -- a good idea, somewhat extensive to implement and the consequences of not having it aren't too high (since manual priority flags can be adjusted instead of doing the operation, and someone seeing the results in Status can see to change these). Moving to v2.0.
